spt/OrangeBox/modules/ClientDLL.cpp

```diff
@@ -90,21 +90,21 @@ void ClientDLL::Hook(const std::wstring& moduleName, HMODULE hModule, uintptr_t
 	ptnNumber = MemUtils::FindUniqueSequence(moduleStart, moduleLength, Patterns::ptnsDoImageSpaceMotionBlur, &pDoImageSpaceMotionBlur);
 	if (ptnNumber != MemUtils::INVALID_SEQUENCE_INDEX)
 	{
-		ORIG_DoImageSpaceMorionBlur = (_DoImageSpaceMotionBlur)pDoImageSpaceMotionBlur;
+		auto ORIG_DoImageSpaceMotionBlur = (_DoImageSpaceMotionBlur)pDoImageSpaceMotionBlur;
 		EngineDevMsg("[client dll] Found DoImageSpaceMotionBlur at %p (using the build %s pattern).\n", pDoImageSpaceMotionBlur, Patterns::ptnsDoImageSpaceMotionBlur[ptnNumber].build.c_str());
 
 		switch (ptnNumber)
-		{
+		{		
 		case 0:
-			pgpGlobals = *(uintptr_t **)(pDoImageSpaceMotionBlur + 132);
+			pgpGlobals = *(uintptr_t **)(pDoImageSpaceMotionBlur + 129);
```

Comment:
Why did you switch these offsets around? Did you switch patterns as well? I would prefer not doing that.